MantidPlot/src/Folder.cpp

```diff
@@ -54,8 +54,7 @@ QStringList Folder::subfolders() {
   QStringList list;
   QObjectList folderList = children();
   if (!folderList.isEmpty()) {
-    QObject *f;
-    foreach (f, folderList)
+    foreach (QObject *f, folderList)
```

Comment:
FWIW, the `foreach` macro is deprecated. I added removing it to the list of maintenance tasks.